code/modules/mob/living/simple_animal/hostile/retaliate/clown.dm

```diff
@@ -292,6 +292,9 @@
 	attack_verb_simple = "awkwardly flail at"
 	loot = list(/obj/item/clothing/mask/gas/clown_hat, /obj/effect/gibspawner/xeno/bodypartless, /obj/item/soap, /obj/effect/gibspawner/generic, /obj/effect/gibspawner/generic/animal, /obj/effect/gibspawner/human/bodypartless, /obj/effect/gibspawner/human)
 
+/mob/living/simple_animal/hostile/retaliate/clown/mutant/slow
+	speed = 20
```

Comment:
does this speed stat still affect this mob if it's controlled by a player?

Reply:
assume so

Reply:
Correct me if I'm wrong, but the speed stat doesn't do anything to the mob, only the player. move_to_delay modifies AI speed, speed modifies player speed.

Reply:
yeah youre right, fucking dumb but ill fix it.